### ISSUE TITLE
Warsaw (and other themes) incompatible with polyglossia

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#221]
Originally reported by: **Anonymous**

----------------------------------------

The following minimal file 
````
\documentclass{beamer}
\usepackage{polyglossia}
\usetheme{Warsaw}%Also with AnnArbor, CambridgeUS, Copenhagen, Darmstadt, EastLansing, Frankfurt, Luebeck, Malmoe,
\begin{document}
\section{sec1}
\begin{frame}\end{frame}
\end{document}
````
produces a warning of the following nature when typeset with XeLaTeX: 
````
** WARNING ** Annotation out of page boundary.
** WARNING ** Current page's MediaBox: [0 0 362.835 272.126]
** WARNING ** Annotation: [161.664 264.278 172.913 273.053]
** WARNING ** Maybe incorrect paper size specified.
````

The same warning also appears with the other themes listed in the comment after `\usetheme` but not with other themes.

I have the texlive 2012 distribution the packages are up to date:
beamer 2011/09/12 development version 3.22
polyglossia 1.2.1

 

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/221

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

This comes down to the font change that happens when `fontspec` is loaded (by `polyglossia`) and how this affects the `headline` template. I will adjust the settings in this area later today (on the wrong PC at present).

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

Increase headline spacing (fixes issue #221)

When LM is in use, there was not enough space reserved here. This fix deals
with the issue, hopefully without breaking too much existing layouts.
